js/calendar.js

```diff
@@ -83,3 +91,153 @@ function refreshCalendar(nextState) {
 			break;
 	}
 }
+
+function loadEvent(event) {
+	let title = "";
+	let date = "";
+	let location = "";
+	let description = "";
+
+	// fetch calendar data
+	if (event) {
+		event.jsEvent.preventDefault(); // don't let the browser navigate
+		let eventData = event.event;
+		let eventDataExtended = eventData.extendedProps;
+
+		// title
+		title = eventData.title;
+
+		// date
+		var dash = " - ";
+		var colon = ":"
+		var space = " ";
+		var allDay = eventData.allDay;
+		var startMinute = ("0" + eventData.start.getMinutes()).slice(-2);
+		var endMinute = ("0" + eventData.end.getMinutes()).slice(-2);
+		var startHour = ("0" + eventData.start.getHours()).slice(-2);
+		var endHour = ("0" + eventData.end.getHours()).slice(-2);
+		var startDay = eventData.start.getDate();
+		var endDay = eventData.end.getDate();
+		var startMonth = eventData.start.getMonth();
+		var endMonth = eventData.end.getMonth();
+		var startYear = eventData.start.getFullYear()
+		var endYear = eventData.end.getFullYear();
+		const months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
```

Comment:
We can declare this outside the function I think, just for clarity. And maybe rename it to `MONTHS` so it's clearer it's a constant.

Reply:
Fixed constant naming and moved to top declarations